package_bag/tests.py

```diff
@@ -0,0 +1,59 @@
+import shutil
+import json
+
+from unittest.mock import patch
+from os import listdir, makedirs, getcwd
+from os.path import isdir, join
+
+from django.test import TestCase
+
+from .routines import DiscoverBags, GetRights, CreatePackage, DeliverPackage
+
+from zorya import settings
+
+# Create your tests here.
+
+bag_fixture_dir = join(settings.BASE_DIR, 'fixtures', 'bags')
+rights_fixture_dir = join(settings.BASE_DIR, 'fixtures', 'rights')
+
+
+class TestPackage(TestCase):
+
+    def setUp(self):
+        self.src_dir = settings.SRC_DIR
+        self.tmp_dir = settings.TMP_DIR
+        self.dest_dir = settings.DEST_DIR
+        for d in [self.src_dir, self.tmp_dir, self.dest_dir]:
+            if isdir(d):
+                shutil.rmtree(d)
+        shutil.copytree(bag_fixture_dir, self.src_dir)
+        for dir in [self.dest_dir, self.tmp_dir]:
+            makedirs(dir)
+        self.discover_bags()
+        self.get_rights()
+        self.create_package()
```

Comment:
This function gets called before each test method below. right now this is actually executing the bulk of your tests in this function, which is non ideal. I've slimmed this down a bit, but you may want to go even further, since in some cases you'll want bags in the src_dir, in some the tmp_dir, and in others the dest_dir. So that last `copytree` line is probably best done in a test function.

```suggestion
    def setUp(self):
        self.src_dir = settings.SRC_DIR
        self.tmp_dir = settings.TMP_DIR
        self.dest_dir = settings.DEST_DIR
        for d in [self.src_dir, self.tmp_dir, self.dest_dir]:
            if isdir(d):
                shutil.rmtree(d)
                makedirs(d)
            else:
                makedirs(d)
        shutil.copytree(bag_fixture_dir, self.src_dir)
```